hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/AbstractRealtimeRecordReader.java

```diff
@@ -85,7 +85,7 @@ public AbstractRealtimeRecordReader(RealtimeSplit split, JobConf job) {
 
   private boolean usesCustomPayload(HoodieTableMetaClient metaClient) {
     return !(metaClient.getTableConfig().getPayloadClass().contains(HoodieAvroPayload.class.getName())
-        || metaClient.getTableConfig().getPayloadClass().contains("org.apache.hudi.OverwriteWithLatestAvroPayload"));
+        || metaClient.getTableConfig().getPayloadClass().contains("org.apache.hudi.common.model.OverwriteWithLatestAvroPayload"));
```

Comment:
Can we use `OverwriteWithLatestAvroPayload.class.getName()` to avoid potential typos and so it will be updated if the location of the class changes?

Reply:
got you, and fixed
Thanks for your suggestion.